src/test/java/seedu/address/model/group/GroupTest.java

```diff
@@ -6,13 +6,14 @@
 import org.junit.jupiter.api.Test;
 
 public class GroupTest {
+    private final Group group1 = new Group("2103T");
+    private final Group group2 = new Group("2101");
+    private final Group group1Duplicate = new Group("2103T");
```

Comment:
Nice! Thanks for making the code more less cluttered by creating one instance to use for multiple tests instead of creating multiple object each time